src/platform/plugins/shared/discover/public/embeddable/get_search_embeddable_factory.tsx

```diff
@@ -98,49 +96,51 @@ export const getSearchEmbeddableFactory = ({
       const fetchWarnings$ = new BehaviorSubject<SearchResponseIncompleteWarning[]>([]);
 
       /** Build API */
-      const { titlesApi, titleComparators, serializeTitles } = initializeTitles(initialState);
+      const titleManager = initializeTitleManager(initialState);
       const timeRange = initializeTimeRange(initialState);
       const searchEmbeddable = await initializeSearchEmbeddableApi(initialState, {
         discoverServices,
       });
       const unsubscribeFromFetch = initializeFetch({
         api: {
           parentApi,
-          ...titlesApi,
+          ...titleManager.api,
           ...timeRange.api,
           savedSearch$: searchEmbeddable.api.savedSearch$,
-          dataViews: searchEmbeddable.api.dataViews,
-          savedObjectId: savedObjectId$,
-          dataLoading: dataLoading$,
-          blockingError: blockingError$,
+          dataViews$: searchEmbeddable.api.dataViews$,
+          savedObjectId$,
+          dataLoading$,
+          blockingError$,
           fetchContext$,
           fetchWarnings$,
         },
         discoverServices,
         stateManager: searchEmbeddable.stateManager,
+        setDataLoading: (dataLoading: boolean | undefined) => dataLoading$.next(dataLoading),
+        setBlockingError: (error: Error | undefined) => blockingError$.next(error),
```

Comment:
I'm curious about these. Is it because their API interfaces are no longer writable?